app/components/articleSearch/components/searchItem/infoList.tsx

```diff
@@ -53,43 +54,42 @@ const InfoList = (props: IInfoListProps) => {
         style={!citedCount ? { display: "none" } : null}
         className={styles.citedButton}
       >
+        <Icon className={styles.citedIconWrapper} icon="CITED" />
         Cited {citedCount}
       </a>
+      <a
+        href={pdfSourceUrl}
+        target="_blank"
+        onClick={() => {
+          trackAndOpenLink("searchItemPdfButton");
+        }}
+        style={!pdfSourceUrl ? { visibility: "hidden" } : null}
+        className={styles.pdfButton}
+      >
+        <Icon className={styles.pdfIconWrapper} icon="PDF_ICON" />
+        PDF
+      </a>
+      <div
+        onClick={() => {
+          copyDOI(DOI);
+        }}
+        style={!DOI ? { visibility: "hidden" } : null}
+        className={styles.copyDOIButton}
+      >
+        DOI : {DOI}
+      </div>
       {/* <span className={styles.explanation}>Cited Paper Avg IF</span>
       <span className={styles.citedPaperAvgIF}>{citedPaperAvgIF}</span>
       <div className={styles.separatorLine} />
       <span className={styles.explanation}>Pltuo Score</span>
       <span className={styles.pltuoScore}>{plutoScore}</span> */}
-      <div className={styles.rightBox}>
-        <a
-          href={pdfSourceUrl}
-          target="_blank"
-          onClick={() => {
-            trackAndOpenLink("searchItemPdfButton");
-          }}
-          style={!pdfSourceUrl ? { visibility: "hidden" } : null}
-          className={styles.pdfButton}
-        >
-          <Icon className={styles.pdfIconWrapper} icon="PDF_ICON" />
-          PDF
-        </a>
-        <div
-          onClick={() => {
-            copyDOI(DOI);
-          }}
-          style={!DOI ? { visibility: "hidden" } : null}
-          className={styles.copyDOIButton}
-        >
-          Copy DOI
-        </div>
-      </div>
     </div>
   );
 };
 
 function copyDOI(DOI: string) {
   const textField = document.createElement("textarea");
-  textField.innerText = DOI;
+  textField.innerText = "https://dx.doi.org/" + DOI;
```

Comment:
is is safe?

Reply:
If doi is null, the doi button does not appear. so I think it is safe. Did you mean it?